sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/SoraMediaChannel.kt

```diff
@@ -223,8 +224,19 @@ class SoraMediaChannel @JvmOverloads constructor(
          * - [PeerChannel]
          *
          * @param mediaChannel イベントが発生したチャネル
+         * @param closeResult 切断結果
          */
-        fun onClose(mediaChannel: SoraMediaChannel) {}
+        fun onClose(mediaChannel: SoraMediaChannel, closeResult: SoraCloseResult?) {}
+
+        /**
+         * Sora との接続が切断されたときに呼び出されるコールバック.
+         *
+         * cf.
+         * - [PeerChannel]
+         *
+         * @param mediaChannel イベントが発生したチャネル
+         */
+        fun onClose(mediaChannel: SoraMediaChannel,) {}
```

Comment:
カンマがついた関係で差分になっていそうです。

```suggestion
        fun onClose(mediaChannel: SoraMediaChannel) {}
```